packages/feathers/src/application.ts

```diff
@@ -93,7 +93,7 @@ export class Feathers<Services, Settings> extends EventEmitter implements Feathe
     }
 
     const protoService = wrapService(location, service, options);
-    const serviceOptions = getServiceOptions(service, options);
+    const serviceOptions = getServiceOptions(protoService);
```

Comment:
Just a note for myself while I review:  Since `wrapService` already calls `getServiceOptions` with the `options`, we don't need it here.